src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -109,10 +112,15 @@ public Task Stats()
             
             return Task.CompletedTask;
         }
-        
-        
+
         [RavenAction("/databases/*/timeseries", "GET", AuthorizationStatus.ValidUser)]
         public Task Read()
+        {
+            return GetRanges();
+        }
+
+        [RavenAction("/databases/*/timeseries/get-ranges", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
=> `/databases/*/timeseries/ranges`

Would be better?

Reply:
Yes, the `GET` is in the HTTP method